src/metatrain/experimental/phace/default-hypers.yaml

```diff
@@ -0,0 +1,43 @@
+architecture:
+  name: experimental.phace 
+  
+  model:
+    nu_max: 3
+    num_message_passing_layers: 2
+    cutoff: 5.0
+    cutoff_width: 1.0
+    num_element_channels: 64
+    radial_basis:
+      mlp: true
+      E_max: 50.0
```

Comment:
should this be defined in terms of `max_radial` instead? I don't think users will understand `E_max`

Reply:
The correspondence is not 1:1

Reply:
I will change it to `max_eigenvalue`, but using some max-radial-like argument removes information

Reply:
fair enough, but I really doubt people will be able to modify this and understand what it does.

> removes information

I'm not clear on what information is lost here?